### ISSUE TITLE
Move from Mute role to discord's new Timeout feature

### ISSUE THREAD

**Issue by comicallybad** [comicallybad/ComicallyBot2.0#20]
**Is your feature request related to a problem? Please describe.**
This will solve some bugs that we currently have with the mute role, such as the mute role not being added/updated to newly created channels. As well, we would not need to create a role anymore, simply use the GuildMember#Timeout() function

**Describe the solution you'd like**
Change all mute code to instead of using a Mute role, use the new discord.js GuildMember#Timeout() function.

**Additional context**
This feature implemented can simply much of the code and get rid of unnecessary roles being created. The GuildMember#Timeout() function is actually really useful after looking further into it.

**Comment by comicallybad** [comment]
just noting: extremely strange behavior. After doing member.timeout(), neither .then() or .catch() work. I have ensured that the line is running, however, it is not timing out the user, and it is not providing any errors as to why it isn't timing out the user as the catch statement is not working. Tried bringing this up to the dickheads in the library discussion channel in the discord.js server and they said that that had nothing to do with the library and I was clearly an idiot that was doing it wrong. So something screwy is happening.
